data/remnant missions.txt

```diff
@@ -495,3 +560,703 @@ mission "Remnant: Technology Available"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have destroyed the Korath ship. You can now return to <planet> to collect your payment."
+	on complete
+		payment 300000
+		dialog "A Remnant military leader thanks you for hunting down the Korath ship <npc>, and gives you the agreed-upon payment of <payment>."
+
+
+
+mission "Remnant: Salvage 5"
+	name "Salvage Delivery"
+	description `Deliver some high value salvage to Caelian.`
+	source "Viminal"
+	cargo "Korath Salvage" 20
+	blocked `Tali would like you to deliver some valuable Korath equipment to Caelian, but you will need 20 tons of space to carry it.`
+	destination "Caelian"
+	to offer
+		or
+			has "Remnant: Salvage 4: done"
+			has "Remnant: Salvage 4: failed"
+		has "Remnant: Learn Sign 3: done"
+	on offer
+		conversation
+			`As you stroll across the tarmac under the great dome you spot Tali heading your way with a flatbed tractor filled with components in tow.`
+			`	"<first>! I'm glad I ran into you. The last raid gave us some particularly good pieces of equipment to study. Could you take these to the weapons laboratory on Caelian?"`
+			choice
+				`	"I could do that."`
+				`	"I'm not interested."`
+					decline
+			`	Tali quickly helps load the cargo onboard your ship. It doesn't appear to be actual whole items anymore, but rather pieces thereof. "They will be expecting you, but you might have to pry them out of their lab to get some help unloading. They can be a bit single-minded when they have something interesting to work on."`
+				accept
+	
+	on complete
+		payment 150000
+		dialog `On Caelian, you are able to find the appropriate research warehouse just off the main landing pad. After a few minutes, a lab technician disentangles herself from a project and brings a wagon and camel team out to the ship to unload the salvage. It strikes you as a bit incongruous that a lab technician is using a graviton repulsion lifter to load cargo from your ship into a wagon pulled by a camel. "If you can, I should have some things to send back to Tali if you could meet me in the cafeteria this afternoon."`
+
+
+
+mission "Remnant: Salvage 6"
+	name "Transport outfits to Tali"
+	description `Deliver some parts to Viminal.`
+	source "Caelian"
+	cargo "Ship parts" 20
+	blocked `The technician you delivered Tali's salvaged parts to would like you to take some finished parts back to Tali, but you will need 20 tons of space to do so.`
+	destination "Viminal"
+	to offer
+		has "Remnant: Salvage 5: done"
+	on offer
+		conversation
+			`You spot the laboratory technician entering the cafeteria, and she approaches you.`
+			`	"Thank you for waiting. I have a shipment of parts ready for Tali, and with the recent increase in attacks she needs them sooner rather than later. Please take them to her."`
+			`	Outside you find a camel pulling a wagon filled with ship parts to be loaded on board <ship>.`
+				accept
+		
+	on complete
+		payment 150000
+		event "Remnant Salvage Available"
```

Comment:
I know I'm the one who moved this, but I hadn't realized there was a delay to when events actually take place (i.e. this event doesn't actually make its changes until you launch), so you can move this back to the on offer so that the Korath outfits are available when Tali actually shows them to you.

Reply:
moved back.